x-pack/plugins/lens/public/editor_frame_service/editor_frame/workspace_panel/workspace_panel.tsx

```diff
@@ -319,6 +363,47 @@ export const InnerVisualizationWrapper = ({
     ]
   );
 
+  if (localState.configurationValidationError) {
```

Comment:
This is also shown during dragging which looks weird (notice the drop visualization overlaying the error message):
<img width="1792" alt="Screenshot 2020-10-29 at 10 37 08" src="https://user-images.githubusercontent.com/1508364/97551748-a57a9480-19d3-11eb-8782-0ebffa84d479.png">

Reply:
It was caused on the same issue as the "building phase" error above. Fixing that, fixed this too.

Reply:
This is still an issue if there's a dimension configured for the break down by, but no metrics and the user starts dragging a field

Reply:
Hm, thinking about it again what should happen is to blur the error message like a rendered chart if the user hovers over the workspace with their field.